pytest_fixtures/core/sys.py

```diff
@@ -24,6 +25,7 @@ def allow_repo_discovery(target_sat):
 @pytest.fixture(autouse=True, scope="session")
 def relax_bfa(session_target_sat):
     """Relax BFA protection against failed login attempts"""
+    logger.info('xxx inside relax_bfa fixture body')
```

Comment:
natural language is typically easy enough to differentiate in logs without trigger characters
```suggestion
    logger.info('inside relax_bfa fixture body')
```

Reply:
sorry, those are my working logs, so i can easily find and remove them when i'm finished :D